ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -35,6 +35,10 @@ tests:
   commands: make unit-test
   container:
     from: src
+- as: functional
+  commands: make functest
+  container:
+    from: src
```

Comment:
The functional tests will execute against a running OpenShift cluster (https://github.com/openshift/release/blob/master/ci-operator/README.md) . You'll need to change this configuration to:

- as: cincinnati-operator-e2e-aws
  commands: |
    make functest
  openshift_installer_src:
    cluster_profile: aws

Reply:
And then regenerate the jobs using "make jobs" and check in the changes under ci-operator/jobs/openshift/cincinnati-operator/.

Reply:
Done.